crypto/mem_sec.c

```diff
@@ -66,6 +66,18 @@ VirtualLock(
 # include <sys/stat.h>
 # include <fcntl.h>
 #endif
+#ifndef HAVE_MADVISE
+# if defined(MADV_DONTDUMP)
```

Comment:
Nit: inconsistency between these two lines (i.e. `#if defined` vs `#ifdef`).

Reply:
(Not blocking approval, though)

Reply:
Note ~ I was originally planning to make this tweak; however, I'm slightly leaning towards leaving as is (unless really desired). The trivial advantage of sticking with the `defined` usage minimizes the mount of changes made in a line needed if additional conditions want to be introduced in the future (i.e. if just appending additional conditional defines).

That being said, switching from an `#ifdef` to a conditional styled one in the future would not be a big issue in terms of something viewing the change. So, if just using `#ifdef` in the source is preferred, I have no problem updating it.

Reply:
It was more, that these are two different methods of doing the same thing... It just looks a bit odd.